cores/arduino/stm32/analog.h

```diff
@@ -47,6 +57,15 @@
  extern "C" {
 #endif
 
+typedef struct {
+  ADC_HandleTypeDef AdcHandle;
```

Comment:
We're making sure that AdcHandle is the first struct inside AdcContext => add a comment to make sure this is always true.